Emby.Naming/Video/VideoListResolver.cs

```diff
@@ -129,35 +151,59 @@ private static List<VideoInfo> GetVideosGroupedByVersion(List<VideoInfo> videos,
                 }
             }
 
-            if (videos.Count > 1)
+            var list = new List<VideoInfo>();
+            if (collectionType.Equals(CollectionType.TvShows, StringComparison.OrdinalIgnoreCase))
+            {
+                var groupedList = mergeable.GroupBy(x => EpisodeGrouper(x.Files[0].Path, namingOptions, collectionType));
+                foreach (var grouping in groupedList)
+                {
+                    list.Add(OrganizeAlternateVersions(grouping.ToList(), grouping.Key.AsSpan(), primary));
+                }
+            }
+            else if (mergeable.Count() > 0)
```

Comment:
```suggestion
            else if (mergeable.Count > 0)
```

Reply:
It's also be nice if comment would start capitalized

Reply:
Implemented